src/verifier/argumentMapper/arguments.ts

```diff
@@ -177,6 +178,18 @@ export const ffiFnMapping: FnMapping<
       return { status: FnValidationStatus.IGNORE };
     },
   },
+  pactffiVerifierSetFailIfNoPactsFound: {
+    validateAndExecute(ffi, handle, options) {
+      if (options.failIfNoPactsFound) {
```

Comment:
The main issue is that now the default behaviour is to _not_ fail if no pacts are found (which is not currently the case, and may be surprising to some users). I'll have to have a think if this is sensible or not, or if we should check if the value is defined and set the value to false.

We may need to consider adding the default somewhere higher up in the stack (e.g. Pact JS)

Reply:
Late Friday mistake on my behalf then. I am more than happy to rework this as you are 100% right. This can pass in true if not provided else the value provided. I'm away from my laptop for the next few days but I can look at fixing in when I'm back

Reply:
All good. I'll add the check now and bring it in - much appreciated!